### DISCUSSION TITLE
The h1 tag cannot be displayed in the article display on the blog homepage.

### DISCUSSION THREAD

**Discussion by chairowell** [vuepress-theme-hope/vuepress-theme-hope#4617] [Q&A] [answered]
I'm a bit confused about this piece of code: .vp-article-excerpt h1 { display: none; }. Why it's written this way?

![image](https://github.com/user-attachments/assets/3c6260b0-7e4f-47b6-b4ba-fce2ff96a33f)

**Comment by Mister-Hope** [comment, marked as answer]
To avoid displaying page title multiple times.

**Reply by chairowell** [reply]
I understand it now. Thank you for your explanation.
